cloud/maintenance.md

```diff
@@ -67,4 +67,47 @@ downtime is usually between 30&nbsp;seconds and 5&nbsp;minutes. We endeavor to
 notify you on email ahead of the upgrade if downtime is required, so that you
 can plan accordingly. However, in some cases, we might not be able to do so.
 
+## Upgrade to a new PostgreSQL version
+Timescale currently supports PostgreSQL&nbsp;12 and 13. You can manually upgrade
+to the next PostgreSQL version in the Timescale Cloud dashboard.
+
+Upgrading to a newer version of PostgreSQL allows you to take advantage of new
+features, enhancements, and security fixes. It also ensures that you are using a
+version of PostgreSQL that is compatible with the newest version of TimescaleDB,
+allowing you to take advantage of everything Timescale has to offer. For more
+information about feature changes between versions, see the
+[PostgreSQL release notes][postgres-relnotes] and
+[TimescaleDB release notes][timescale-relnotes].
+
+<highlight type="warning">
+Your Timescale Cloud service is unavailable for use until the upgrade is
+complete. Upgrading can take up to several hours to complete, so we recommend
+that you plan ahead, and upgrade during a time with low usage.
+</highlight>
+
+If you want to try out the upgrade before you run it on your production system,
+you can create a fork of your service, and attempt the upgrade on the fork
+before you start. This can give you a good idea of what happens during an
+upgrade, and how long it might take in your use case. For more infomration about
+creating a fork of your service, see the [Operations
+section][operations-forking].
+
+<procedure>
+
+### Upgrading to a new PostgreSQL version
+1.  In the Timescale Cloud console, navigate to `Services` and click the service
+    you want to adjust.
+1.  Navigate to the `Operations` tab, and go to the `Maintenance` section.
+1.  If a new PostgreSQL version is available, click the `Upgrade` button, and
+    confirm that you are ready to start the upgrade. Your Timescale Cloud
+    service is unavailable for use until the upgrade is complete.
+1.  When the upgrade is finished, your service automatically resumes normal
+    operations. If the upgrade is unsuccessful, the service returns to the state
+    it was in before you started the upgrade.
```

Comment:
If the upgrade fails, is there a message that appears and are there any details about how to resolve the issues if any?